### PR TITLE
Don't skip sleep when recording cassettes

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -82,10 +83,12 @@ def peach_with_index(*)
         VCR.insert_cassette("#{self.class.metadata[:description]}/all")
 
         # avoid polling idle time by overriding sleep
-        module Kernel
-          alias :old_sleep :sleep
-          def sleep(n)
-            n
+        if skip_sleep
+          module Kernel
```
Comment (yenkins) [inline spec/spec_helper.rb]: Has no descriptive comment (IrresponsibleModule)

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -101,8 +104,10 @@ def sleep(n)
         load('pmap.rb') if self.class.metadata[:vcr]
 
         # reload sleep method
-        module Kernel
-          alias :sleep :old_sleep
+        if skip_sleep
+          module Kernel
```
Comment (yenkins) [inline spec/spec_helper.rb]: Has no descriptive comment (IrresponsibleModule)

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -82,10 +83,12 @@ def peach_with_index(*)
...
 
         # avoid polling idle time by overriding sleep
-        module Kernel
-          alias :old_sleep :sleep
-          def sleep(n)
-            n
+        if skip_sleep
+          module Kernel
+            alias :old_sleep :sleep
+            def sleep(n)
```
Comment (yenkins) [inline spec/spec_helper.rb]: Has the parameter name 'n' (UncommunicativeParameterName)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/147/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/988/ : SUCCESS in 52s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1257/ : FAILURE in 51s

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/148/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/989/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1258/ : SUCCESS in 49s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/149/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1408/ : FAILURE in 55m 19s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1175/ : UNSTABLE in 16m 28s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/151/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/991/ : SUCCESS in 2m 13s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1260/ : SUCCESS in 2m 19s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/152/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1409/ : SUCCESS in 55m 04s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1176/ : UNSTABLE in 15m 35s

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/154/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1410/ : SUCCESS in 57m 05s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1177/ : UNSTABLE in 15m 57s

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/64/ : FAILURE in 13s
- gooddata-ruby-pronto-code-review-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED
- gooddata-ruby-integration-tests-zuul : SKIPPED
- gooddata-ruby-lcm-integration-tests-zuul : SKIPPED

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/155/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1411/ : ABORTED in 16m 14s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1178/ : FAILURE in 33s

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/156/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1413/ : ABORTED in 56s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1181/ : FAILURE in 37s

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/157/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1414/ : SUCCESS in 54m 26s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1182/ : FAILURE in 33s

**Comment by panjan** [comment]
tests passed: https://travis-ci.org/panjan/gooddata-ruby/builds/385716612

**Pull request merged**